react/features/recording/components/Recording/styles.native.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import { createStyleSheet } from '../../../base/styles';
+
+/**
+ * The styles of the React {@code Components} of the feature recording.
+ */
+export default createStyleSheet({
+    header: {
+        flex: 0,
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        paddingTop: 15,
```

Comment:
can we use constants instead of `15` ? We used to reference BoxModel from 'base/styles' for such things

Reply:
Shouldn't I use BoxModel if value will be used in a lot of places? Why do we want constant?

Reply:
I see the same value being used in a number of places here. You could define a module level constant to `BoxModel.margin *1.5`. Why do it? To change it everywhere in a single go, if we ever need to.

Reply:
OK :)

Reply:
Actually I'd prefer to not reference `BoxModel.margin` because this is very specific layout and I'm worried that we can accidentally change it trough `BoxModel.margin`. OK?

Reply:
I don't see how is it specific. It's just a dialog like other dialog we have in the app. And the point with referencing `BoxModel.margin` is to keep them consistent to some extend at least. That's the intention to have it changed when `BoxModel.margin` would change, so nothing to be worried about here.